jbmc/src/java_bytecode/java_local_variable_table.cpp

```diff
@@ -467,7 +467,7 @@ static java_bytecode_convert_methodt::method_offsett get_common_dominator(
     const auto &dominator_nodeidx=
       dominator_analysis.cfg.entry_map.at(v->var.start_pc);
     const auto &this_var_doms=
-      dominator_analysis.cfg[dominator_nodeidx].dominators;
+      dominator_analysis.dominated_by(dominator_nodeidx);
```

Comment:
I think the name `dominated_by` is the opposite of what we want? This should be the set of dominators of `dominator_nodeidx`